src/Angular.js

```diff
@@ -634,12 +633,14 @@ function isDate(value) {
  * @kind function
  *
  * @description
- * Determines if a reference is an `Array`. Alias of Array.isArray.
+ * Determines if a reference is an `Array`.
  *
  * @param {*} value Reference to check.
  * @returns {boolean} True if `value` is an `Array`.
  */
-var isArray = Array.isArray;
+function isArray(arr) {
+  return arr instanceof Array || Array.isArray(arr);
```

Comment:
`Array.isArray(arr)` is unreachable here, since by definition any array exotic object is `instanceof Array`.